config.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * MozIoT Gateway Configuration.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+'use strict';
+
+exports.THINGS_PATH = '/things';
+exports.NEW_THINGS_PATH = '/new_things';
+exports.ADAPTERS_PATH = '/adapters';
+exports.ACTIONS_PATH = '/actions';
+exports.STATIC_PATH = 'static';
+exports.DEBUG_PATH = '/debug';
```

Comment:
There is also a config directory. Currently, it just has the gpio config. Perhaps this should be a file in the config directory, and have index.js